content_script.js

```diff
@@ -134,6 +89,91 @@ class HTMLSerializer {
   }
 
   /**
+   * Takes an html element, and populates this object's fields with the
+   * appropriate attribute names and values.
+   *
+   * @param {Element} element The Element to serialize.
+   * @private
+   */ 
+  processAttributes(element) {
```

Comment:
This is a class function right? Why does it look like it's declared as a static global?